buildpack_integration_test.go

```diff
@@ -135,15 +151,34 @@ func testBuildpackIntegration(t *testing.T, context spec.G, it spec.S) {
 }
 
 func archiveToDaemon(path, id string) error {
-	skopeo := pexec.NewExecutable("skopeo")
+	tmpDir := os.TempDir()
```

Comment:
We could (and probably should) refactor this to [`t.TempDir()`](https://pkg.go.dev/testing#T.TempDir). But that would require refactoring this method to accept `t` and I think it's worth doing as a separate refactor.